### PR TITLE
[RW-8654][risk=no]  Polish Demographic Survey Flow - Add question number to error message

### PR DESCRIPTION
**Pull request by NehaBroad**
Screen shot existing using disability 
Screen Shot 2022-07-27 at 10 49 25 AM

Updated : use difficulty  rather than Disability 
Screen Shot 2022-07-29 at 2 36 37 PM

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

**Review by jmthibault79 — COMMENTED** [review]
Code looks good and it works great, thanks!

I'd like to see a few changes however:
1. can we order the error messages by question number?
2. we can improve the wording of some of the error messages too, like "disability errands" (this can be a different PR)

File: ui/src/app/components/demographic-survey-v2.tsx [hunk 1]
```diff
@@ -30,20 +45,36 @@ import { WithSpinnerOverlayProps } from './with-spinner-overlay';
...
+  DISABILITY_CONCENTRATING,
+  DISABILITY_WALKING,
+  DISABILITY_DRESSING,
+  DISABILITY_ERRANDS,
+  EDUCATION,
+  DISADVANTAGED,
 ];
 
+export const questionsIndex = {
+  ethnicCategories: 1,
```
[the remark this answers is not in the document]
Comment (evrii) [reply ui/src/app/components/demographic-survey-v2.tsx:63]: Did not know that you could do that. That is a neat feature:
https://stackoverflow.com/a/11508490

File: ui/src/app/utils/constants.tsx [hunk 2]
```diff
@@ -4,3 +4,21 @@ export const DEMOGRAPHIC_SURVEY_V2_PATH = '/demographic-survey';
 
 export const DEMOGRAPHIC_SURVEY_SESSION_KEY =
   'demographic-survey-banner-dismissed';
+
+export const ETHNIC_CATEGORIES = 'ethnicCategories';
+
```
[the remark this answers is not in the document]
Comment (NehaBroad) [reply ui/src/app/utils/constants.tsx:9]: no just copy paste i'll fix this

File: ui/src/app/components/demographic-survey-v2.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (NehaBroad) [reply ui/src/app/components/demographic-survey-v2.tsx:63]: Nice! will try this

[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/components/demographic-survey-v2.tsx:63]: I actually did not know either - I just assumed there was a way.  Good find!

**Comment by NehaBroad** [comment]
Error message ordering - Done
Tried updating the error message as well

**Review by jmthibault79 — APPROVED** [review]
Looks good, works locally.

Regarding the error message update - looks like this approach will work!  Let's agree on the preferred text with Moira.

**Review by jmthibault79 — APPROVED** [review]

**Pull request merged**